nmostesting/Config.py

```diff
@@ -97,6 +97,11 @@
 # The mock DNS server port cannot be modified from the default of 53.
 PORT_BASE = 5000
 
+# A valid unicast/multicast IP address on the local network which media streams can be sent to. This will be passed
+# into Sender configuration when testing IS-05.
+UNICAST_STREAM_TARGET = "127.0.0.1"
+MULTICAST_STREAM_TARGET = "239.10.53.5"
```

Comment:
Would defaulting each of these to one of the addresses in the appropriate reserved-for-testing/documentation ranges work?

Unicast:
https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
https://tools.ietf.org/html/rfc5737

Multicast:
https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xhtml#multicast-addresses-11
https://tools.ietf.org/html/rfc5771#section-9.2

Reply:
Yes, now that they're configurable it makes much more sense to use these.